proposals/NNNN-android-transport-overhaul.md

```diff
@@ -0,0 +1,491 @@
+
+#### ForceOnConnect Removal
+
+The `ForceOnConnect` extra will no longer be used so the logic for it can be removed. This piece of integration has always been extra clutter and very error prone. It's removal from necessity will make the library more stable and less developer error prone.
+
+#### AOA and TCP Multiplexing
+
+A new USB and TCP transport discussed later, will be implemented into the `SdlRouterService`. This will allow all apps to connect through the USB and TCP transports when it is available. 
+
+###### TCP Single SDL Port Assumption
+A current assumption about the TCP transport is that the SDL core module will only expose a single port for all apps to connect through for a secondary transport. 
+
+The ability will be added to modify this in the future so that the router service can hold multiple transports of the same type, but for unless it is required the feature can be implemented at a latter time. 
+
+
+### Simultaneous Transport Modifications 
+
+#### MultiplexTransport Config Updates
+
+The config will be updated to hold a few extra settings
+
+1. Primary Transport prioritized list - Ordered list of primary transports this app supports
+    -  **Default order:** USB, Bluetooth
+2. Secondary Transport prioritized list - Ordered list of secondary transports this app supports
+    -  **Default order:** WiFi, USB, Bluetooth
+3. High bandwidth required flag - flag for those apps that will require at least one available transport that can support the high bandwidth needs. Default will be false.
+
+In practice, developers will only need to set the high bandwidth required flag. However, the setting of primary and secondary transports allows developers to test against certain transports as well as ensures complete control of how they intend for their app to work.
+
+
+#### Transport Switching
+
+Pushing all the transports into the router service we are able to maintain connections through proxy cycle events. This means the router service will automatically connect to transports as they become available and maintain that connection. Depending on each apps primary transport allowance and priority, they will be connected on that basis. 
+
+If a transport disconnects that is considered to be an apps primary transport, the app will check the remaining connected transport types against it's prioritized primary transport list. If a transport is found to be acceptable, the app will will perform a proxy cycle event and rebind with the router service and thus connect over the acceptable transport. 
```

Comment:
Please update to "...an app's primary..." (add apostrophe) and "...its prioritized... (remove apostrophe).